cmd/minikube/cmd/delete.go

```diff
@@ -145,6 +146,32 @@ func deleteContainersAndVolumes(ctx context.Context, ociBin string) {
 	}
 }
 
+// kicbaseImage fetches images based on container label
+func kicbaseImages(ctx context.Context, ociBin string) ([]string, error) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return nil, nil
+	}
+	delLabel := fmt.Sprintf("%s=%s", oci.CreatedByLabelKey, "true")
+	names, err := oci.ListImagesByContainerLabel(ctx, ociBin, delLabel)
+	if err != nil {
+		return nil, err
+	}
+	return names, nil
+}
+
+// printDeleteImagesCommand prints command which remove images
+func printDeleteImagesCommand(ociBin string, imageNames []string) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return
+	}
+
+	if len(imageNames) > 0 {
+		out.Styled(style.Command, `{{.ociBin}} rmi {{.images}}`, out.V{"ociBin": ociBin, "images": strings.Join(imageNames, "")})
+	} else {
+		out.Styled(style.Command, `{{.ociBin}} rmi <KICBASE IMAGE>`, out.V{"ociBin": ociBin})
```

Comment:
I'm not sure if having this line makes the most sense. Assuming the user is only using Docker or Podman, the output is going to look something like:
```
docker rmi kicbase:latest
podman rmi <KICBASE IMAGE>
```
We could omit the line, but would look odd if there was no KIC image for either and the output was empty. Maybe it would make sense to replace with something like `No docker kicbase images to remove`?

Reply:
I improved that and now message is only printed when images are found.
But this is a little bit tricky as well when user will run `minikube delete --purge -all` twice.
Message will be printed only for the first time. During the second run we won't find any images because containers no more exist. Is this sufficient solution?

Reply:
Yeah, I looked at your changes and this is fine